clang/lib/Driver/Driver.cpp

```diff
@@ -3607,6 +3610,24 @@ class OffloadingActionBuilder final {
 
     void appendLinkDependences(OffloadAction::DeviceDependences &DA) override {}
 
+    void addDeviceLinkDependencies(OffloadDepsJobAction *DA) override {
+      for (unsigned I = 0; I < ToolChains.size(); ++I) {
+        // Register dependent toolchain.
+        DA->registerDependentActionInfo(
+            ToolChains[I], /*BoundArch=*/StringRef(), Action::OFK_OpenMP);
+
+        if (!ToolChains[I]->getTriple().isSPIR()) {
```

Comment:
Just to check my understanding: this is possible due to the fact that we're combining SYCL and OpenMP offloading in a single compilcation?

Reply:
No, this is not directly related to SYCL. OpenMP offload target is defined by the user in command line and it can be set to spir64 for example.